index.js

```diff
@@ -37,6 +38,24 @@ module.exports.filter = shouldCompress
 
 var cacheControlNoTransformRegExp = /(?:^|,)\s*?no-transform\s*?(?:,|$)/
 
+/**
+ * @const
+ * whether current node version has brotli support
+ */
+var hasBrotliSupport = process.versions ? 'brotli' in process.versions : false
```

Comment:
It looks like the code coverage only shows one of these branches are ever run. Under what Node.js versions would the other branch run?